### DISCUSSION TITLE
Do not Buttons have tab focus indicator? is it intended behavior?

### DISCUSSION THREAD

**Discussion by EminQasimov** [DouyinFE/semi-design#583] [Q&A] [answered]
I mean when I want to tab navigation, I didn't notice any focus style indicator on buttons. it is not good for a11y.
between this https://semi.design/en-US/input/form tab crashed when I select TImePicker from "Supported Fields example collection
" section

**Comment by shijiatongxue** [comment, marked as answer]
> I mean when I want to tab navigation, I didn't notice any focus style indicator on buttons. it is not good for a11y. between this https://semi.design/en-US/input/form tab crashed when I select TImePicker from "Supported Fields example collection " section

Sorry for the late reply. Button focus style has beed added. Accessibility is an important part of Semi, and we are gradually improving the accessibility experience of components and sites. Check issue #205 see more information.

![image](https://user-images.githubusercontent.com/26477537/172753579-057832d6-c3b4-4290-8ea1-b53dd754df50.png)

![image](https://user-images.githubusercontent.com/26477537/172753535-7a500489-f5fe-4ac7-912e-7e607d9ff7fe.png)
